acceptance-tests/upgrade/update_and_upgrade_mongodb_test.go

```diff
@@ -149,6 +154,13 @@ var _ = Describe("UpgradeMongoTest", Label("mongodb"), func() {
 			By("checking previous data still accessible")
 			got = app.GET("%s/%s/%s", databaseName, collectionName, documentNameOne)
 			Expect(got).To(Equal(documentDataOne))
+
+			By("deleting the backing service") // Can't delete with broker as TF version too high
+			cmd := exec.Command("az", "cosmosdb", "delete", "--resource-group", metadata.ResourceGroup, "--name", fmt.Sprintf("csb%s", serviceInstance.GUID()), "--yes")
```

Comment:
Non-blocking: is it worth putting this in an after each as it is just cleanup?  just as it is not really behaviour we are testing.